### ISSUE TITLE
how to plot some data  in real time?

### ISSUE THREAD

**Issue by cherishyuan** [alandefreitas/matplotplusplus#405]
I am trying to plot some data  in real time using cpp. However, the real-time plotting  doesn't seem to be working.  Please tell me if it is supported. I have not seen any relevant examples in the documentation. Similar to the following

![tEqqt](https://github.com/alandefreitas/matplotplusplus/assets/96415301/a17e2ebd-08a4-4697-b600-0a5c30add380)

**Comment by alandefreitas** [comment]
Yes. In terms of the API, it's technically supported. In practice, the only backend we have is Gnuplot, and it's terrible for real-time plotting, especially at high FPS. We don't really have the human resources to implement these other backends either.
